src/Qir/Common/cmake/qir_cmake_include.cmake

```diff
@@ -39,3 +39,79 @@ macro(target_source_from_qir target_name source_file)
         ${OBJFILE}
     )
 endmacro()
+
+
+#===============================================================================
+# Common flags
+
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDEBUG")
+set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -DDEBUG")
+
+if (NOT WIN32)
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fsanitize=undefined -fsanitize=float-divide-by-zero -fsanitize=unsigned-integer-overflow -fsanitize=implicit-conversion -fsanitize=local-bounds -fsanitize=nullability")
```

Comment:
nitpick: Is there a reason to not just combine this and the lines below into a single long string with newlines? Something like:
```
set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} \
  -fsanitize=undefined \
  -fsanitize=float-divide-by-zero \
 ...
")
```